src/main/java/teammates/ui/webapi/request/AccountCreateRequest.java

```diff
@@ -0,0 +1,50 @@
+package teammates.ui.webapi.request;
+
+import teammates.common.util.FieldValidator;
+
+/**
+ * The basic request of creating new account.
```

Comment:
I think you can drop the basic since it isn't a base request like FeedbackQuestionBasicRequest